### PR TITLE
15 contact page

### LINKED ISSUE #15: Contact page
This page should contain a **form** with **submit** a **reset button** and the fields:
Name
Email
Message

Also it should have social networks icons, e.g Linkedin and github, with links to the team page.

### PR DESCRIPTION
**Pull request by nadianogues**
This pull request resolves #15. The contact table was add and the database was populated.

### CODE REVIEW

File: bd/blog.sql [hunk 1]
```diff
@@ -1,10 +1,8 @@
-CREATE DATABASE  IF NOT EXISTS `blog` /*!40100 DEFAULT CHARACTER SET latin1 */;
-USE `blog`;
```
Comment (rodrigoAMF) [inline bd/blog.sql]: Please add again these two lines.

File: src/app/contact/contact.component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@Component({
+  selector: 'app-contact',
+  templateUrl: './contact.component.html',
+  styleUrls: ['./contact.component.scss']
+})
+export class ContactComponent implements OnInit {
+
+  constructor() 
+  { 
+    // (document).ready(function() {
```
Comment (rodrigoAMF) [inline src/app/contact/contact.component.ts]: Remove these lines commented

File: src/app/contact/contact.component.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    //   $("#linkedin-nadia-button").click(function() 
+    //   {
+    //       window.location = "https://www.linkedin.com/in/nadianoguesalmeida/";
+    //   });
+    // });
+
+  }
+
+  redirectLinkedinNadia($event){    
+    console.log("Save button is clicked!", $event);
```
Comment (rodrigoAMF) [inline src/app/contact/contact.component.ts]: Do we really need to log this message here?

File: src/app/menu/menu.component.ts [hunk 4]
```diff
@@ -1,14 +1,12 @@
...
 
 @Component({
   selector: 'app-menu',
   templateUrl: './menu.component.html',
-  styleUrls: ['./menu.component.scss']
+  styleUrls: ['./menu.component.scss'],
 })
 export class MenuComponent implements OnInit {
 
-  constructor() { }
```
Comment (rodrigoAMF) [inline src/app/menu/menu.component.ts]: Add again the empty constructor

File: src/app/contact/contact.component.ts (hunk 3 above)
Comment (nadianogues) [reply src/app/contact/contact.component.ts]: No

**Pull request merged**